meetuppizza/tests.py

```diff
@@ -1,6 +1,23 @@
 from django.test import TestCase
+from django.contrib.auth.models import User
+from django.test import Client
+from meetuppizza.forms import RegistrationForm
+from django.contrib import auth
+
+import pdb
+
 
 class Test(TestCase):
```

Comment:
This is a bad name for this class. What is it actually testing? Do all the test methods belong together, or should they be split into several conceptually related classes?